Src/VimMac/RelativeLineNumbers/CocoaExtensions.cs

```diff
@@ -11,6 +11,7 @@
 using CoreGraphics;
 using CoreText;
 using Foundation;
+using ObjCRuntime;
```

Comment:
Why is this necessary? Did types move around to a new namespace?

Reply:
Yes. System.nfloat got moved to ObjCRuntime.nfloat